data/json/recipes/recipe_deconstruction.json

```diff
@@ -7708,4 +7708,60 @@
     "qualities": [ { "id": "HAMMER", "level": 2 }, { "id": "SAW_M", "level": 2 } ],
     "components": [ [ [ "steel_lump", 7 ], [ "steel_chunk", 2 ] ] ]
   }
+  {
+    "result": "meat",
+    "type": "uncraft",
+    "activity_level": "NO_EXERCISE",
+    "time": "20 s",
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "meat_scrap", 10 ] ] ]
+  },
+  {
```

Comment:
Is this extra meat uncraft necessary?

Reply:
No, must have made a mistake when I was copy pasting my code in.  Removed both of those.